qdrant-landing/content/documentation/cloud/authentication.md

```diff
@@ -101,3 +101,14 @@ var client = new QdrantClient(
   apiKey: "<paste-your-api-key-here>"
 );
 ```
+
+```go
+import "github.com/qdrant/go-client/qdrant"
+
+client, err := qdrant.NewClient(&qdrant.Config{
```

Comment:
The `err` variable is declared but not used for brevity in all the snippets.
If any user copies the snippets, the compiler will enforce handling it.